pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -1388,4 +1391,15 @@ public void registerOwnedBundles() {
             }
         });
     }
+
+    @Override
+    public void close() throws IOException {
```

Comment:
Can we remove the 'throws' clause?
We can also move to AutoCloseable instead of Closeable